server/src/main/java/org/eclipse/openvsx/web/WebConfig.java

```diff
@@ -34,7 +34,10 @@ public void addCorsMappings(CorsRegistry registry) {
             var authorizedEndpoints = new String[] {
                 "/user/**",
                 "/logout",
-                "/api/*/*/review/**"
+                "/api/-/namespace/create",
+                "/api/*/*/review/**",
+                "/api/*",
```

Comment:
You can remove `/api/-/namespace/create` and `/api/*` now that everything related to the dashboard is in `/admin/**`.